.travis.yml

```diff
@@ -106,7 +106,7 @@ matrix:
     - name: "c: linux tsan"
       language: cpp
       env:
-          - INTLTEST_OPTS="utility/MultithreadTest rbbi/RBBIMonkeyTest format/CalendarLimitTest"
+          - INTLTEST_OPTS="utility/MultithreadTest"
```

Comment:
Did you move the old tests? I don't see that.  If you do move them, how about adding a top-level "thread" category parallel to utility.

Reply:
The tests outside of the MultithreadTest directory are threaded primarily to reduce the elapsed runtime of long-running functional tests for some service or another. They run too long to conveniently run under the thread sanitizer, which is really slow. 

Bottom line, I think they're fine where they are, with the service they're testing.